app/common/styles.css

```diff
@@ -1,40 +1,31 @@
-/*.col-sm-6 {
-background-color: #00BCD4;
-padding-bottom: 35px;
-}
-
-h3 {
-padding-top: 2px;
-padding-bottom: 10px;
-font-size: 20px;
-}
-
-.padding {
-padding-top: 10px !important;
-}
-
-.contstyle {
-background-color: #00BCD4;
-}
-
-#tableId {
-overflow-x: auto;
-}
-
-
-
-.notReverse:after {
-content: '\25bc'; // BLACK DOWN-POINTING TRIANGLE}
-.reverse:after {
-    content: '\25b2'; // BLACK UP-POINTING TRIANGLE
-}
-#searchDiv {
-    float: right;
-}
-#btnDiv {
-    float: left;
-}
-#notFound {
-    text-align: center;
+.headingcolor {
+    color: rgb(74, 74, 74);
+    font-size: 16px;
+    font-family: Roboto;
```

Comment:
if you are using roboto font in entire project. then simply apply Roboto to * so it gets applied to all elements